nautobot/extras/templates/extras/job_detail.html

```diff
@@ -115,6 +115,17 @@
                 <td>{{ object.enabled | render_boolean }}</td>
                 <td></td>
             </tr>
+            <tr>
+                <td>Job Results</td>
+                <td>
+                    {% if object.job_results.all %}
```

Comment:
```suggestion
                    {% if object.job_results.exists %}
```

Reply:
Thanks, that was copy pasta from location children.